server/tests/permission-groups-cli.rs

```diff
@@ -0,0 +1,80 @@
+use assert_cmd::prelude::*;
+use cosmicverge_shared::permissions::{AccountPermission, Permission, Service};
+use database::schema::{PermissionGroup, Statement};
+use std::{env, process::Command, str::FromStr};
+
+#[tokio::test]
+async fn permission_groups() -> anyhow::Result<()> {
+    database::test_util::initialize_exclusive_test().await;
```

Comment:
Integration tests never run in parallel anyway. They are always in their own process too.

Reply:
That's good to know, so while the guard isn't important in this situation, the initialization code is still critical, and the method in which it initializes it (full wipe) is important.

Reply:
Sounds good.

Reply:
This is good to know, but this call still needs to be here. It wipes the database clean and ensures the database is initialized, both which are required by the idea of what an "exclusive test" needs.